android/app/build.gradle

```diff
@@ -208,14 +208,14 @@ android {
         dev {
             minSdkVersion rootProject.ext.minSdkVersion
             targetSdkVersion rootProject.ext.targetSdkVersion
-            resValue "string", "app_name", "BetterSocial Dev"
+            resValue "string", "app_name", "HelioDev"
             applicationIdSuffix ".dev"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The application name has been changed from "BetterSocial Dev" to "HelioDev" for the dev build variant. This change is consistent with the PR title and should not cause any issues.



<!-- This is an auto-generated comment by OSS CodeRabbit -->